app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class RecipesController < ApplicationController
+  def show
+    @recipe = Recipe.find(params[:id])
+    @ingredients = @recipe.ingredients
```

Comment:
I would probably just call `@recipe.ingredients` in the view. That wouldn't be considered too much logic for the view and then has the added benefit of slimming down this controller.